### PR TITLE
merge code in pooling

### PR DESCRIPTION
**Pull request by mli**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_convolutional-neural-networks/pooling.md [hunk 1]
```diff
@@ -284,19 +256,20 @@ Below, we will concatenate arrays `X` and `X+1`
...
-#@tab pytorch
+
 X = torch.cat((X, X + 1), dim=1)
 X
 ```
 
 ```{.python .input}
 #@tab tensorflow
-X = tf.reshape(tf.stack([X, X+1], axis=0), (1, 2, 4, 4))
+X = tf.reshape(tf.stack([X, X+1], 0), (1, 2, 4, 4))
```
Comment (terrytangyuan) [inline chapter_convolutional-neural-networks/pooling.md:272]: This can be d2l.reshape instead?

Comment (mli) [reply chapter_convolutional-neural-networks/pooling.md:272]: quite different to the preview cell, so not necessary to use d2l here.

Comment (AnirudhDagar) [reply chapter_convolutional-neural-networks/pooling.md:268]: I thought the idea is to now use `d2l` as much as possible, even if the code blocks are not intended to be merged.
Anyways they will be changed back using reverse aliases, so why not use them everywhere possible for more consistency.

**Comment by mli** [comment]
Job d2l-en/PR-1200/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1200/

**Pull request merged**